src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/NavMenu.razor.cs

```diff
@@ -16,7 +16,7 @@ public partial class NavMenu
                      new NavLink { Name= "Button", Key = "Button", Url = "/components/buttons", TargetType = NavLinkTargetType.Self },
                      new NavLink { Name= "Checkbox", Key = "Checkbox", Url = "#" },
                      new NavLink { Name= "Choice", Key = "Choice", Url = "#" },
-                     new NavLink { Name= "Label", Key = "Label", Url = "#" },
+                     new NavLink { Name= "Label", Key = "Label",  Url = "/components/labels", Target = "_Self" },
```

Comment:
the Target parameter should not be set here, because this value causes a hard refresh of the application